src/common/runtime/src/global.rs

```diff
@@ -36,6 +37,9 @@ pub struct RuntimeOptions {
     pub write_rt_size: usize,
     /// The number of threads to execute the runtime for global background operations.
     pub bg_rt_size: usize,
+    /// The number of threads to execute the runtime for heartbeat between meta
+    /// and this node
+    pub hb_rt_size: usize,
```

Comment:
I just follow the `bg_rt_size` naming pattern but it's ok to use a better name for readability.

Reply:
Just changed my mind because we can also use `rt` for `runtime`. It's better to keep consistent naming pattern, like `bg`/`hb`/`rt`. Once we feel it's necessary to change, we change it in one batch.

Reply:
`bg` and `rt` are well-known abbreviations in CS but I'm afraid that `hb` isn't. We don't need to keep using abbreviations here since we also have `write_rt_size` and `read_rt_size`, not `w_rt_size` and `r_rt_size`.

What's more, maybe using `runtime` is clear than `rt`, from my perspective.

Reply:
Is it necessary to allow configuring the runtime size of the heartbeat runtime? It's hard to imagine that a node needs to occupy more than one core to send heartbeats to the meta server......